Utils/azure-explorer-common/test/java/com/microsoft/tooling/IntegrationTestBase.java

```diff
@@ -284,28 +305,52 @@ private String applyRegex(String text) {
         }
         return text;
     }
-    
+
     private static Boolean IsMocked() {
-        // TODO: support un mocked mode
+        String keyValue=System.getProperty("isMockedCase");
```

Comment:
IsMocked is originally to test the currently running environment. So it is same as IS_MOCKED.  It seems you are using this method as ismockedtestcase right?